about/complying_with_licenses.rst

```diff
@@ -16,6 +16,12 @@ distribute the software (and derivative projects, including games made with it).
 Your game or project can have a different license, but it still needs to comply
 with the original one.
 
+.. note::
+
+    This section covers compliance with licenses from a user perspective.
+    If you are interested in licence compliance as a contributor, please see
+    the last two paragraphs of :ref:`Best Practices for Engine Contributors <doc_best_practices_for_engine_contributors>`
```

Comment:
Instead of referring to "the last two paragraphs" we should add an anchor next to the appropriate part and link to that.

Reply:
Ah ok, will do.  Thanks for the feedback!

Reply:
I've added an anchor to last paragraph of "Beast Practices for Engine Contributors" page and linked to it.  I also reworded the highlighted snippet.